src/components/ui4/CardUi4.tsx

```diff
@@ -65,48 +77,59 @@ export const CardUi4 = (props: Props) => {
     triggerHaptic('impactLight')
   })
 
-  return (
-    <TouchableHighlight
-      accessible={false}
-      onPress={handlePress}
-      onLongPress={handleLongPress}
-      disabled={handlePress == null && handleLongPress == null}
-      underlayColor={theme.touchHighlightUi4}
-      style={styles.cardContainer}
-    >
-      <>
-        <LinearGradient {...theme.cardBackgroundUi4} {...underlayBackground} style={[StyleSheet.absoluteFill, styles.backgroundFill]}>
-          {underlayForeground}
-        </LinearGradient>
-        {icon == null ? null : <View style={styles.iconContainer}>{icon}</View>}
-        <SectionView>{children}</SectionView>
-        {onClose == null ? null : (
-          <TouchableOpacity style={styles.cornerContainer} onPress={handleClose}>
-            <AntDesignIcon color={theme.iconTappableAltUi4} name="close" size={theme.rem(1.25)} />
-          </TouchableOpacity>
-        )}
-        {overlay == null ? null : <View style={styles.overlayContainer}>{overlay}</View>}
-      </>
-    </TouchableHighlight>
+  const underlay = (
+    <LinearGradient {...(underlayBackground || theme.cardBackgroundUi4)} style={[StyleSheet.absoluteFill, styles.backgroundFill]}>
```

Comment:
Instead of doing `style={[ ... ]}` to smash the two styles together here, you can do `backgroundFill: { ...StyleSheet.absoluteFillObject, borderRadius: ... }` down in the CSS area. That will bake all the styles together into one object. The `absoluteFillObject` is the same as `absoluteFill`, but as a JavaScript object that can be splatted out.